domain/sysvar_cache.go

```diff
@@ -145,6 +145,10 @@ func (svc *SysVarCache) RebuildSysVarCache(ctx sessionctx.Context) error {
 }
 
 // checkEnableServerGlobalVar processes variables that acts in server and global level.
+// This is required because the SetGlobal function on the sysvar struct only executes on
+// the initiating tidb-server. There is no current method to say "run this function on all
+// tidb servers when the value of this variable changes". If you do not require changes to
+// be applied on all servers, use a getter/setter instead! You don't need to add to this list.
```

Comment:
This clarifies the use of this function, hoping users will not add additional cases unnecessarily. I am not sure if in future it makes sense to add functionality to specifically address this so a `SetGlobal` can execute code on all servers.